### ISSUE TITLE
Add license

### ISSUE THREAD

**Issue by agg23** [baxysquare/mister_kun#1]
Could you add the proper license for this art?

**Comment by birdybro** [comment]
I believe credit for the logo and ~~character~~ goes to Conrad Fenech, and he typically just wants credit, but you should check with him on Facebook messaging to be sure.

This repo owner made a higher res version of Conrad Fenech's artwork.

**Comment by baxysquare** [comment]
I'm happy to to add a license and give credit to Conrad Fenech as recommended.

I've always worked under the assumption that this artwork was covered by the same license as the MiSTer project.  It was simply a vectorized redraw so the logo could be used for other purposes.  For the last two years it has served that exact purpose and nothing more.  It's hosted as its own project so it didn't waste space in any of the mainline project repositories.  If it needs to go somewhere else instead, I'm happy to do whatever is needed.

[In the license](https://github.com/MiSTer-devel/MkDocs_MiSTer/blob/main/LICENSE), it gives logo credit to "HeWhoIsRed".  But the [ReadMe](https://github.com/MiSTer-devel/MkDocs_MiSTer/blob/main/README.md) makes the distinction that Conrad is the original author.

**Update:** I was able to chat with Conrad and confirm that he did the design of the "Logotype" with the outlined typography, **but not for the MiSTer Kun mascot**.  Conrad was commissioned to do the design and the rights were officially transferred to @sorgelig, so I need to ask him for permission.  So Sorg, may I have retroactive permission for this repository?

Conrad did not know who created the MiSTer Kun mascot and has no idea who @HeWhoIsRed is. Any insight into these mysteries would be appreciated.

**Comment by Hewhoisred** [comment]
Yep, I am indeed the creator of the original logo.  The logo is a gift to the Mister community, and my intention is for it to be used/remixed/whatever'd as people wish.  That said, I do appreciate a simple accreditation when it is possible and appropriate.

**Comment by agg23** [comment]
@Hewhoisred, sounds like you might want something like Creative Commons - Attribution Required (CC BY).

![](https://www.wur.nl/upload_mm/e/f/c/0527e0fb-5053-4045-8ee0-95d91f127f76_Creative%20commons%20licences.jpg)

I was going to suggest that you might not want it to allow commercial, but hardware designs might use the mascot, and if they weren't open source, they'd be in violation.

**Comment by baxysquare** [comment]
Awesome @Hewhoisred!  I'm happy to recognize you and Conrad as the original designers in the license.  Thanks for creating the design.  I always wanted to know who created it and ask about the inspiration for the mascot.  I'd love to hear the story of how it came to be.

Here's a draft of a license and attribution, working on the assumption @sorgelig will give permission and further details on use that will be contained in the license.

```
--------------------------------
License & Attribution for MiSTer-Kun Mascot Artwork & Logotype
--------------------------------
The MiSTer-Kun mascot was originally created by GitHub user "HeWhoisRed" for the MiSTer FPGA Project and the design is considered a gift to the MiSTer community. The intention is for the mascot to be used/remixed/whatever'd as people wish. A simple accreditation is appreciated when it is possible and appropriate.

The MiSTer logotype with thick outline and rounded typography was created by Conrad Fenech and commissioned by Alexey Melnikov (hereby known as "Sorgelig").  Sorgelig has given permission to use the logotype on this repository and full terms will be added here.  Conrad Fenech is to receive attribution as part of the original agreement.

The icons and logos in the MiSTer-Kun GitHub Repository are remastered versions of the mascot and logotype and adhere to the above license terms.  The remastered mascot was created by GitHub user "BAXYsquare". The remastered logotype was created by Javier Rodas and originally shared on Facebook, then combined with the mascot to create three variations of the combined "logo" design.
```

**Comment by sorgelig** [comment]
I don't see point of this repository.
MiSTer logo belongs to MiSTer project and cannot be freely used anywhere just because some one decided to put.

**Comment by baxysquare** [comment]
@sorgelig I'd like to formally apologize, explain and talk about next steps. This repository was created was so the community had access to high resolution logos for use in the MiSTer project itself.  [For two years it has served this purpose without issue or concern](https://misterfpga.org/viewtopic.php?p=21775#p21775).  

It was created as its own separate repository because I had no idea which repository in MiSTer-devel it should go in and I did not want to add unnecessary bloat to the existing codebase.  

If you'd like me to transfer this repository to the MiSTer-devel organization, I would like to do so.  If you'd rather I submit the files to an existing repository, let me know where you want them and I'll submit them as a Pull Release.  

If neither of these options are satisfactory, let me know and I will delete the project in order to comply with your wishes.  My intent was to contribute to the project and nothing more.  I'm sincerely sorry and want to make this right.

**Comment by birdybro** [comment]
The high resolution logo would be useful to keep, I used this rendition of it on the MkDocs site. Maybe these could just live on the MkDocs repo under assets? They are low footprint vector graphics.

**Comment by baxysquare** [comment]
Thanks @birdybro for the offer, and I'm happy to get all the files moved over to MkDocs if that's what @sorgelig wishes.

As far as I can tell, [it's also not too hard to transfer the repository to MiSTer-devel](https://docs.github.com/en/repositories/creating-and-managing-repositories/transferring-a-repository) and letting it be on its own. But I've never done anything like this and perhaps it's more difficult that it appears to be.

I honestly have no preference on where the files are hosted.  I simply want to make sure they go where project administator(s) want them to go.  From there, I will amend all the posts on the [MiSTer FPGA forum](https://misterfpga.org/) so people can find them accordingly.

**Comment by SegaSnatcher** [comment]
> I don't see point of this repository. MiSTer logo belongs to MiSTer project and cannot be freely used anywhere just because some one decided to put.

Do you mean literally the MiSTer logo or the Cat Kun image too?    I was under the impression the actual Kitty-Kun image was free for anyone to use and not something currently owned by anyone for exclusive use.

**Comment by birdybro** [comment]
MiSTer Logo = Property of sorgelig, was a commissioned job by Conrad Fenech.

MiSTer-kun = Totally open and free for anyone to use as per @Hewhoisred

**Comment by SegaSnatcher** [comment]
> MiSTer Logo = Property of sorgelig, was a commissioned job by Conrad Fenech.
> 
> MiSTer-kun = Totally open and free for anyone to use as per @Hewhoisred

Yeah, that is what I assumed, thanks.

**Comment by baxysquare** [comment]
Technically, there are only three files that infringe, based on this definition.  They are the files that have the prefix "combined_logo_mascot." They contain the [artwork by Javier Rodas on the MiSTer Facebook page](https://www.dropbox.com/s/gym92g5i8eauhdw/MISTER%20FPGA%20Logo.svg?dl=0&fbclid=IwAR0TrZi1tpoQxnI_2zch-y4bWovwNpb8rNV9uHwKCArVnMv-W2SE3SvRy_Q) and available for [download on the MiSTer FPGA forum](https://misterfpga.org/download/file.php?id=8172).  All I did was add MiSTer-kun to the left, right or above the logo.

If you'd prefer I simply remove those three files and add a license file that works for @Hewhoisred, I can go that route. EDIT: I can also move these three files to MKDocs and call it a day.  What do you think I should do?

**Comment by baxysquare** [comment]
I went ahead and added the infringing files via [Pull Release on the MkDocs site](https://github.com/MiSTer-devel/MkDocs_MiSTer/pull/46) and removed them from this repository for compliance.  I have added a [license document](https://github.com/baxysquare/mister_kun/blob/main/LICENSE) that contains an edited version of the draft license from above.  I also updated the ReadMe so that people can find the Combined Logo Mascot files on MkDocs.

By removing these files, I consider this issue closed for now.  If @Hewhoisred would like to formally adopt a CC license, I will update the license to reflect their wishes. The moved files are sufficiently covered by the MkDocs license, so once the pull release is merged, we should be good to go.
